code/modules/reagents/chemistry/holder.dm

```diff
@@ -266,6 +266,8 @@
 	return amount
 
 /datum/reagents/proc/metabolize(mob/living/carbon/C, can_overdose = FALSE, liverless = FALSE)
+	if(HAS_TRAIT(C, NO_REAGENTS))
```

Comment:
This seems like it should break something but I don't know what

Reply:
it doesnt break anything, ive tested it

Reply:
Can't wait for the followup PRs